### PR TITLE
Fix red light detection for suite_carla

### PR DESCRIPTION
**Pull request by emailweixu**

### CODE REVIEW

File: alf/environments/carla_sensors.py [hunk 1]
```diff
@@ -955,6 +957,15 @@ def _get_traffic_light_waypoints(self, traffic_light):
                     wpx = next_wp
                 else:
                     break
+            if wpx.transform.location.distance(
+                    area_loc) >= self.RED_LIGHT_ENFORCE_DISTANCE - 2:
```
Comment (hnyu) [inline alf/environments/carla_sensors.py:961]: What is special about -2 here?

File: alf/environments/carla_sensors.py [hunk 2]
```diff
@@ -640,6 +640,10 @@ def _to_numpy_loc(loc: carla.Location):
     return np.array([loc.x, loc.y, loc.z], dtype=np.float)
 
 
+def _to_carla_loc(loc):
```
Comment (hnyu) [inline alf/environments/carla_sensors.py:643]: This function seems not used anywhere.

**Review by hnyu — CHANGES_REQUESTED** [review]

File: alf/environments/carla_sensors.py (hunk 1 above)
Comment (emailweixu) [reply alf/environments/carla_sensors.py:961]: distance between the center of the car and its tail

File: alf/environments/carla_sensors.py (hunk 2 above)
Comment (emailweixu) [reply alf/environments/carla_sensors.py:643]: I used it when I wrote debugging code. It doesn't hurt to keep it.

**Review by hnyu — APPROVED** [review]

**Pull request merged**